net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+		<url>/pkg_edit.php?xml=snmptt.xml</url>
+	</menu>
+	<service>
+		<name>snmptt</name>
+		<rcfile>snmptt.sh</rcfile>
+		<executable>snmptt</executable>
+		<description>SNMPTT Daemon</description>
+		<custom_php_service_status_command>
+			<![CDATA[
+				""; $output=""; exec("/bin/pgrep -anf snmptt", $output, $retval); $rc=(intval($retval) == 0)
+			]]>
+		</custom_php_service_status_command>
+	</service>
+	<tabs>
+		<tab>
+			<text>General</text>
+			<url>/pkg_edit.php?xml=snmptt.xml</url>
+			<active />
+		</tab>
+		<tab>
+			<text>snmptt.conf</text>
+			<url>/pkg_edit.php?xml=snmptt_conf.xml</url>
+		</tab>
+	</tabs>
+	<fields>
+		<field>
+			<name>General Settings</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Enable</fielddescr>
+			<fieldname>snmpttenabled</fieldname>
+			<description>Enable SNMPTT service.</description>
+			<type>checkbox</type>
+		</field>
+		<field>
+			<fielddescr>SNMPTT System Name</fielddescr>
+			<fieldname>snmptt_system_name</fieldname>
+			<description>
+				<![CDATA[
+					Name of this system for $H variable.  If blank, system name will be the computer's<br/>
+					hostname via Sys::Hostname.<br/>
+				]]>
+			</description>
+			<default_value></default_value>
+			<type>input</type>
+			<size>50</size>
+		</field>
+		<field>
+			<fielddescr>Operation Mode</fielddescr>
+			<fieldname>mode</fieldname>
+			<description>
+				<![CDATA[
+					standalone: snmptt called from snmptrapd.conf<br/>
+					daemon: snmptrapd.conf calls snmptthandle<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>standalone</default_value>
+			<options>
+				<option><name>standalone</name><value>standalone</value></option>
+				<option><name>daemon</name><value>daemon</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Multiple Event</fielddescr>
+			<fieldname>multiple_event</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to allow multiple trap definitions to be executed for the same trap.<br/>
+					Set to 0 to have it stop after the first match.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>DNS Enable</fielddescr>
+			<fieldname>dns_enable</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable DNS resolution.<br/>
+					Set to 1 to enable DNS resolution.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Strip Domain</fielddescr>
+			<fieldname>strip_domain</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to enable the use of FQDN (Fully Qualified Domain Names). If a host name is<br/>
```

Comment:
Why not put short descriptions of these into the name of the options below? That would be more user-friendly.

For example:
`<option><name>Use Fully Qualified Domain Names (0)</name><value>0</value></option>`
`<option><name>Remove all domain names (1)</name><value>1</value></option>`
`<option><name>Remove domain names from strip_domain_list (2)</name><value>2</value></option>`